lms/static/scripts/frontend_apps/components/LaunchErrorDialog.tsx

```diff
@@ -212,6 +212,21 @@ export default function LaunchErrorDialog({
         </ErrorModal>
       );
 
+    case 'canvas_page_not_found_in_course':
+      return (
+        <ErrorModal
+          {...defaultProps}
+          title="Hypothesis couldn't find the page in the course"
+        >
+          <p>This might have happened because:</p>
+
+          <ul className="px-4 list-disc">
+            <li>The page has been deleted from Canvas</li>
+            <li>The course was copied from another course</li>
+          </ul>
+        </ErrorModal>
```

Comment:
I see we link to our KB docs for other errors. Let's make sure we have an issue to add the same here once available.